internal/armada/repository/job.go

```diff
@@ -18,15 +18,17 @@ import (
 	"github.com/G-Research/armada/pkg/api"
 )
 
-const jobObjectPrefix = "Job:"             // {jobId}            - job protobuf object
-const jobStartTimePrefix = "Job:StartTime" // {jobId}            - map clusterId -> startTime
-const jobQueuePrefix = "Job:Queue:"        // {queue}            - sorted set of jobIds by priority
-const jobLeasedPrefix = "Job:Leased:"      // {queue}            - sorted set of jobIds by lease renewal time
-const jobSetPrefix = "Job:Set:"            // {jobSetId}         - set of jobIds
-const jobClusterMapKey = "Job:ClusterId"   //                    - map jobId -> cluster
-const jobRetriesPrefix = "Job:Retries:"    // {jobId}            - number of retry attempts
-const jobClientIdPrefix = "job:ClientId:"  // {queue}:{clientId} - corresponding jobId
-const keySeparator = ":"
+const (
+	jobObjectPrefix    = "Job:"          // {jobId}            - job protobuf object
+	jobStartTimePrefix = "Job:StartTime" // {jobId}            - map clusterId -> startTime
+	jobQueuePrefix     = "Job:Queue:"    // {queue}            - sorted set of jobIds by priority
+	jobLeasedPrefix    = "Job:Leased:"   // {queue}            - sorted set of jobIds by lease renewal time
+	jobSetPrefix       = "Job:Set:"      // {jobSetId}         - set of jobIds
+	jobClusterMapKey   = "Job:ClusterId" //                    - map jobId -> cluster
+	jobRetriesPrefix   = "Job:Retries:"  // {jobId}            - number of retry attempts
+	jobClientIdPrefix  = "job:ClientId:" // {queue}:{clientId} - corresponding jobId
+	keySeparator       = ":"
+)
 
 // Number of jobs queried from Redis at a time in IterateQueueJobs.
 const queueResourcesBatchSize = 20000
```

Comment:
I'm really confused about when the autofixer chose to combine const sets and when it didn't...

Reply:
Looks like it groups by type?

Reply:
Looks like comments turn it off for grouping.